api/src/main/java/de/tesis/dynaware/grapheditor/window/PanningWindow.java

```diff
@@ -10,8 +10,10 @@
 import javafx.collections.ObservableList;
 import javafx.event.Event;
 import javafx.event.EventHandler;
+import javafx.geometry.HPos;
```

Comment:
Is that import really needed? I only see usages on `Pos`?

Reply:
Yeah, we can remove it. Thank you again.

Reply:
Thanks for your work!